configs/quic.toml

```diff
@@ -0,0 +1,24 @@
+# This configuration is an example to show Retina in "offline" mode, and
+# reads from the /traces/quic.pcap files. 
+# 
+# See https://stanford-esrg.github.io/retina/retina_core/config/index.html
+# for configuration options.
+
+main_core = 0
+nb_memory_channels = 4
+
+[mempool]
+    capacity = 4004
```

Comment:
Any reason that the mempool capacity is 4004 and not a power of 2? This file also may not be needed at all since it is just `offline.toml` with the pcap target changed.

Reply:
No reason. I can remove this from being tracked if that's helpful? It doesn't add any value.

Reply:
Yes, let's remove it then.